lib/Debugger.ProtocolHandler/RuntimeImpl.cpp

```diff
@@ -315,6 +316,38 @@ namespace JsDebug
         return true;
     }
 
+    JsValueRef JsObjectToString(JsValueRef value, JsValueRef* error)
```

Comment:
I would rather name this JsJSONStringfyObject or something like that. Because toString on the object will supposed to return [object Object]

Reply:
done